Doc/library/inspect.rst

```diff
@@ -665,6 +665,11 @@ function.
    Accepts a wide range of Python callables, from plain functions and classes to
    :func:`functools.partial` objects.
 
+   If *follow_wrapped* is ``False`` *callable* will not be unwrapped
+   (``callable.__wrapped__`` will not be used to unwrap decorated callables).
+
+   If *skip_bound_arg* is ``False``, keep ``self`` parameter in a signature.
```

Comment:
```python
    @classmethod
    def from_callable(cls, obj, *,
                      follow_wrapped=True, bound_arg=False,
                      globals=None, locals=None, eval_str=False):
        """Constructs Signature for the given callable object."""
        return _signature_from_callable(obj, sigcls=cls,
                                        follow_wrapper_chains=follow_wrapped,
                                        skip_bound_arg=not bound_arg,
                                        globals=globals, locals=locals, eval_str=eval_str)
```

Ok! I wanted to have the same name, but since all other objects do not expose `skip_bound_arg`, it is just an implementation detail. So, we can show the better name.